llvm/test/CodeGen/CHERI-Generic/RISCV64/landingpad-non-preemptible.ll

```diff
@@ -112,7 +112,7 @@ declare dso_local void @__cxa_end_catch() local_unnamed_addr addrspace(200)
 ; Note: RISC-V uses DW_EH_PE_udata4, so the 0xc marker uses 4 bytes instead of 1
 ; CHECK-NEXT:  [[SMALL_CS_DIRECTIVE:(\.byte)|(\.word)]] 12     # (landing pad is a capability)
 ; Note: the following line should not be using _Z8do_catchv, but a local alias
-; CHECK-NEXT:  .chericap  .L_Z8do_catchv$local+(.Ltmp2-.Lfunc_begin0)   #     jumps to .Ltmp2
+; CHECK-NEXT:  .chericap  %code(.L_Z8do_catchv$local+(.Ltmp2-.Lfunc_begin0))   #     jumps to .Ltmp2
```

Comment:
Now that there is a specifier and a separate relocation it I wonder if we can stop using .Lfunc_begin0 and just do `%code(.L_Z8do_catchv$local+(.Ltmp2-.L_Z8do_catchv$local)). Or maybe even use something like `%code(.Ltmp2)` and have the object writer or linker use the closest function symbol for the bounds?